### PR TITLE
Add Spanish translations

### PR DESCRIPTION
**Pull request by diaverso**
**When merged this pull request will:**
- Add translations to all functions into Spanish.

### CODE REVIEW

File: addons/ai/stringtable.xml [hunk 1]
```diff
@@ -54,6 +59,7 @@
             <Portuguese>Precisão da Mira</Portuguese>
             <Japanese>照準精度</Japanese>
             <Korean>조준 정확도</Korean>
+            <Spanish>Precisión al apuntar</spanish>
```
Comment (mharis001) [inline addons/ai/stringtable.xml:62]: Invalid closing tag. Should be `</Spanish>` (capital S).

Comment (diaverso) [reply addons/ai/stringtable.xml]: Fix

File: addons/ai/stringtable.xml [hunk 2]
```diff
@@ -112,6 +122,7 @@
             <Japanese>探知距離</Japanese>
             <Korean>색적 거리</Korean>
             <Polish>Odległość Spottingu</Polish>
+            <Spanish>Distancia de localización</Spanish
```
Comment (neilzar) [inline addons/ai/stringtable.xml:125]:
```suggestion
            <Spanish>Distancia de localización</Spanish>
```

File: addons/ai/stringtable.xml [hunk 3]
```diff
@@ -135,115 +147,134 @@
...
             <Polish>Błąd w celowaniu</Polish>
             <Korean>조준 오류</Korean>
             <Japanese>照準エラー</Japanese>
+            <Spanish>Error al apuntar</Spanish>
         </Key>
         <Key ID="STR_ZEN_AI_AnimChange">
             <English>Anim Change</English>
             <Polish>Zmiana animacji</Polish>
             <Japanese>アニメーション変更</Japanese>
+            <Spanish>Cambio de animación</spanish>
```
Comment (neilzar) [inline addons/ai/stringtable.xml:171]:
```suggestion
            <Spanish>Cambio de animación</Spanish>
```

**Review by neilzar — CHANGES_REQUESTED** [review]

File: addons/ai/stringtable.xml (hunk 3 above)
Comment (diaverso) [reply addons/ai/stringtable.xml]: Fix

File: addons/ai/stringtable.xml (hunk 2 above)
Comment (diaverso) [reply addons/ai/stringtable.xml]: Fix

**Review by CreepPork — APPROVED** [review]

**Review by mharis001 — APPROVED** [review]

**Review by neilzar — APPROVED** [review]

**Comment by mharis001** [comment]
Thank you for translating.

**Pull request merged**